src/controls/TeamChannelPicker/TeamChannelPicker.tsx

```diff
@@ -48,21 +49,24 @@ export const TeamChannelPicker: React.FunctionComponent<ITeamChannelPickerProps>
   const picker = React.useRef<IBasePicker<ITag>>(null);
   const { serviceScope } = props.appcontext;
   const { getTeamChannels } = useTeams(serviceScope);
-  const {
-    renderItemStylesSingle,
-    renderItemStylesMulti,
-    pickerStylesMulti,
-    pickerStylesSingle,
-    renderIconButtonRemoveStyles,
-  } = useTeamChannelPickerStyles(theme);
   const {
     onSelectedChannels,
     selectedChannels,
     itemLimit,
     label,
     styles,
+    themeVariant,
   } = props;
 
+  const {
+    renderItemStylesSingle,
+    renderItemStylesMulti,
+    pickerStylesMulti,
+    pickerStylesSingle,
+    renderIconButtonRemoveStyles,
+    componentClasses
+  } = useTeamChannelPickerStyles(themeVariant);
+
   const pickerSuggestionsProps: IBasePickerSuggestionsProps = {
     suggestionsHeaderText: "Suggested Team Channels",
     noResultsFoundText: "No channels found",
```

Comment:
@joaojmendes can you update to use loc strings please

Reply:
done